synapse/handlers/user_directory.py

```diff
@@ -125,18 +125,22 @@ def handle_local_profile_change(self, user_id, profile):
         """
         # FIXME(#3714): We should probably do this in the same worker as all
         # the other changes.
-        yield self.store.update_profile_in_user_dir(
-            user_id, profile.display_name, profile.avatar_url, None,
-        )
+        is_support = yield self.store.is_support_user(user_id)
+        if not is_support:
```

Comment:
comment would be good here. why is this dependent on whether the user is a support user?